spec/models/bundle_context_spec.rb

```diff
@@ -1,15 +1,18 @@
-require 'rails_helper'
-
 RSpec.describe BundleContext, type: :model do
-  context "validation" do
-    let(:user) { User.new(sunet_id: "Jdoe")}
-    subject(:bc) { BundleContext.new(project_name: "SmokeTest",
-                                     content_structure: 1,
-                                     bundle_dir: "spec/test_data/bundle_input_g",
-                                     staging_style_symlink: false,
-                                     content_metadata_creation: 1,
-                                     user: user) }
+  subject(:bc) do
+    BundleContext.new(
+      project_name: "SmokeTest",
+      content_structure: 1,
+      bundle_dir: "spec/test_data/bundle_input_g/",
+      staging_style_symlink: false,
+      content_metadata_creation: 1,
+      user: user
+    )
+  end
 
+  let(:user) { User.new(sunet_id: "Jdoe") }
+
+  context "validation" do
     it "is not valid unless it has all required attributes" do
       expect(BundleContext.new).not_to be_valid
       expect(bc).to be_valid
```

Comment:
so if we're using "subject" aren't lines like this written as
```
expect(subject).to be_valid
```

or perhaps an even shorter version:

```
is_expected.to be_valid
```

https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/predicate-matchers

Reply:
https://github.com/sul-dlss/pre-assembly/issues/242

Reply:
By my understanding, no, named subjects are preferred specifically to allow them to be referenced by their name.  Otherwise there is no point naming them.